src/Symfony/Component/Routing/RouteCollectionBuilder.php

```diff
@@ -61,21 +61,28 @@ public function __construct(LoaderInterface $loader = null)
      */
     public function import($resource, $prefix = '/', $type = null)
```

Comment:
I think there is nothing to fix here. Calling `import` does not imply that you were using a glob. However, the previous implementation made this assumption and always added the direction as a resource, even for non-globs, I don't think this is something we want.